### ISSUE TITLE
How to modify URL link ?

### ISSUE THREAD

**Issue by tarocjsu** [AlexSciFier/neonlink#89]
How to modify URL link ? look like only can modify the Name ? Description ? but not the URL link.

**Comment by AlexSciFier** [comment]
Yes it is. The logic of the application is that you cannot change the URL, because if you edit the URL, it means that it is a completely different link. All you can do is remove the old link and add a new one.

**Comment by julichan** [comment]
Actually I'd like to propose a revamp on the link creation/edition for later. I tried some dashboard apps lately and saw they proposed the following features:
- Picking site icon if possible (we already have that but there are instances where the site is inaccessible from the server and i think this should be performed by client and the icon uploaded to the server. Also, i think we should not propose that feature to be launched automatically cause it makes one go crazy to lose a title or description they written themself. )
- propose sets of icon, there are lots of projects for that and some of these solutions may be really easy to implement:
Favicon - Automatically fetch an apps icon from its Favicon or logo image
Icon Packs - Use any icon from [font-awesome](https://fontawesome.com/icons), [simple-icons](https://simpleicons.org/) or [material icons](https://github.com/Templarian/MaterialDesign)
Emoji - Any valid emoji can be used as an icon
URL - Pass the URL of any valid image in to have it fetched and rendered
Local - Store custom images locally and reference by filename
